.azure-pipelines/templates/run-tests.yml

```
@@ -6,14 +6,21 @@ parameters:
   benchmark: false
   latest_metrics: false
   repo: ''
+  coverage: true
 
 steps:
-- ${{ if eq(parameters.test, 'true') }}:
+- ${{ if and(eq(parameters.test, 'true'), eq(parameters.coverage, 'true') }}:
   - script: ddev test --cov --junit ${{ parameters.check }}
     displayName: 'Run Unit/Integration tests'
     env:
       CODECOV_TOKEN: $(CODECOV_TOKEN)
 
+- ${{ if and(eq(parameters.test, 'true'), not(eq(parameters.coverage, 'true')) }}:
+    - script: ddev test --junit ${{ parameters.check }}
```

Should we remove --junit from here if coverage isn't set?
```suggestion
    - script: ddev test ${{ parameters.check }}
```

---

`junit` should be independent of coverage, it outputs the test results in an xml format that CI can read from.